src/utils/AccurateMassSearch.cpp

```diff
@@ -107,13 +107,12 @@ class TOPPAccurateMassSearch :
     Param p = AccurateMassSearchEngine().getDefaults();
     registerTOPPSubsection_("db", "Database files which contain the identifications");
     registerInputFileList_("db:mapping", "<file(s)>", ListUtils::toStringList<std::string>(p.getValue("db:mapping")), p.getDescription("db:mapping"), true, false, {"skipexists"});
-    setValidFormats_("db:mapping", {"tsv"});
+    setValidFormats_("db:mapping", {"sqlite"});
```

Comment:
is your plan to keep both tsv and sqlite? WIP?

Reply:
yes WIP